zone/main.cpp

```diff
@@ -398,7 +398,7 @@ int main(int argc, char **argv)
 		return 1;
 	}
 
-	guild_mgr.LoadGuilds();
+	//guild_mgr.LoadGuilds();
```

Comment:
Why is this commented here ?

Reply:
I see you have it later in zone.cpp now. We can remove it here if not needed

Reply:
Aye.  I moved it to remove the needless sending of guild data from world to zones, when they are not booted and loaded.  Now, when a zone actually loads, it will pull the most recent data from the db, and then it will start to receive updates from world.
Updated.